boards/arm/pinetime_devkit0/key_out.c

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2021 Casper Meijn <casper@meijn.net>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <drivers/gpio.h>
+#include <logging/log.h>
+
+LOG_MODULE_REGISTER(pine64_pinetime_key_out);
+
+#define KEY_OUT_NODE DT_PATH(key_out)
+
+#if DT_NODE_HAS_STATUS(KEY_OUT_NODE, okay)
```

Comment:
Do you really intend to allow users to disable the node? In my previous review, I was suggesting removing this too, or at least that's what I meant to suggest. If you really want to let people turn off this node, this is fine. Just confirming.

Reply:
Well, this driver is needed to make the sample applications work. If an application wants to do something smart with the button (only enable during polling for example), then the application can disable this node and create it's own driver.

I don't know whether that is a realistic use-case, but that was my thinking.